### ISSUE TITLE
Add css-grid support to merge-styles

### ISSUE THREAD

**Issue by redoz** [microsoft/fluentui#6067]
**Describe the feature that you would like added**
Currently (to the best of my knowledge) `gridTemplateAreas` has to be specified like this:
`gridTemplateAreas: '"title controls"\n"subtitle controls"',`

Which is not the nicest syntax in my opinion, it would be nicer if it took an array of string, like so
`gridTemplateAreas: ['title controls', 'subtitle controls'],`

**What component or utility would this be added to**
`merge-styles`

**Have you discussed this feature with our team, and if so, who**
No.

**Additional context/screenshots**

**Comment by dzearing** [comment]
doable, but nice to have.

Similar to other scenarios. Like `content: '""'`

**Comment by dzearing** [comment]
But, I might also argue... it would be even better to have a `Grid` component which abstracts this in props. That way we can support a more standard solution, and perhaps abstract a solution which works with IE11. (fyi, gridTemplateAreas does not work in IE11, if that's in your support matrix.)

**Comment by redoz** [comment]
@dzearing I don't think you're wrong regarding a `Grid` component, but it would still be nice if `merge-styles` had better support for emitting literal strings.

**Comment by dzearing** [comment]
https://medium.com/@elad/supporting-css-grid-in-internet-explorer-b38669e75d66

For reference. We will see what can be done here. 

Even right now, we're a little blocked using css grid because we will, at minimum, translate `display`:

From:
```css
display: grid;
```

to:
```css
display:-ms-grid;
display:grid;
```

**Comment by dzearing** [comment]
We need to be tactical and address this soonish before `Grid` work starts. This should unblock cases.
